<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Resize</class>
 <widget class="QDialog" name="Resize">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>263</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Resize and Reposition</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>46</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Left Border</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>67</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Vertical Seperation</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>24</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Top Border</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>89</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Max Length</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>155</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Label x position</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>111</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Box Width</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>178</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Input x position</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>133</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Horizontal Seperation</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>202</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Label Width</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>225</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Input Seperation</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="topSpin">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>19</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>change the top side of the border bewteen the mainwindow and the first boxes</string>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="leftSpin">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>41</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>change the left side of the border bewteen the mainwindow and the first boxes</string>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="vSpin">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>63</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>change the vertical seperation between the boxes</string>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="maxSpin">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>85</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>change the limit of the length of each column of boxes</string>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="widthSpin">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>108</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>change the width of the boxes</string>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="inputxSpin">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>176</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Change where within the box along the x axis the input control is</string>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="hSpin">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>131</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Chagne the horizontal seperation between the boxes</string>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="labelxSpin">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>154</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Change where along the x axis of the box the label is</string>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="inputSepSpin">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>223</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Change the gap between the input controls</string>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="labelWidthSpin">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Change the width of the labels</string>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
  </widget>
  <widget class="QPushButton" name="ok">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>267</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>267</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="apply">
   <property name="geometry">
    <rect>
     <x>187</x>
     <y>267</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
